src/animation/AnimationUtils.js

```diff
@@ -2,13 +2,13 @@ import { Quaternion } from '../math/Quaternion.js';
 import { AdditiveAnimationBlendMode } from '../constants.js';
 
 // same as Array.prototype.slice, but also works on typed arrays
-function arraySlice( array, from, to ) {
+function arraySlice( array, from, to = array.length ) {
 
 	if ( isTypedArray( array ) ) {
 
 		// in ios9 array.subarray(from, undefined) will return empty array
 		// but array.subarray(from) or array.subarray(from, len) is correct
-		return new array.constructor( array.subarray( from, to !== undefined ? to : array.length ) );
```

Comment:
I think it's best here to check if latest iOS version handle `array.subarray(from, undefined)` correctly. If so, we can remove the default parameter.

According to my understanding, this was only an issue with WebKit on iOS9 but I suggest to double-check to be sure.

Reply:
i meant if *patching the TypedArray.prototype.slice* is now not revelant... then its safe to remove `arraySlice` from codebase :D

Reply:
Compatibility table: https://caniuse.com/mdn-javascript_builtins_typedarray_slice

It looks safe to remove the helper. If for some reasons somebody has issue with a removal, apps can still polyfill `TypedArray.slice()`.